manifests/service/data.pp

```diff
@@ -26,7 +26,7 @@
 
     package { $package_name :
         ensure  => $package_ensure,
-        require => [ Class['lightblue::yumrepo::lightblue'], Class['lightblue::eap'] ],
+        require => [ Class['lightblue::yumrepo::lightblue'], Class['lightblue::eap'], Class['lightblue::service::plugin::rdbms'] ],
```

Comment:
This forces use of rdbms.  Because configuration is done via jcliff this is not needed anyway because it requires eap to be deployed first.  The profile setup for deployment should just include the rdbms class and dependencies will be right.